Gemfile

```diff
@@ -12,8 +12,8 @@ version2x = (RUBY_VERSION =~ /^2\.\d/)
 # https://github.com/ged/ruby-pg/blob/master/History.rdoc
 # https://bitbucket.org/ged/ruby-pg/wiki/Home
 
-# pg >= 1.0.0 doesn't work with Rails at the moment. It's a Rails bug.
 gem "pg"
+gem "psych", "~> 3"
```

Comment:
Though, what's psych for here?

Reply:
Without this restriction I was getting the following error when running tests:
```
└[$]› bundle exec rake
rake aborted!
Psych::BadAlias: Cannot load database configuration:
Unknown alias: default
/Users/maneyko/.rvm/gems/ruby-2.4.6@pg_saurus/gems/psych-4.0.1/lib/psych/visitors/to_ruby.rb:430:in `visit_Psych_Nodes_Alias'
/Users/maneyko/.rvm/gems/ruby-2.4.6@pg_saurus/gems/psych-4.0.1/lib/psych/visitors/visitor.rb:30:in `visit'
/Users/maneyko/.rvm/gems/ruby-2.4.6@pg_saurus/gems/psych-4.0.1/lib/psych/visitors/visitor.rb:6:in `accept'
/Users/maneyko/.rvm/gems/ruby-2.4.6@pg_saurus/gems/psych-4.0.1/lib/psych/visitors/to_ruby.rb:35:in `accept'
/Users/maneyko/.rvm/gems/ruby-2.4.6@pg_saurus/gems/psych-4.0.1/lib/psych/visitors/to_ruby.rb:345:in `block in revive_hash'
```

Reply:
👌